chroma_core/models/server_profile.py

```diff
@@ -19,14 +19,16 @@ class ServerProfile(models.Model):
     ui_name = models.CharField(max_length=50, help_text="String, human readable name")
     ui_description = models.TextField(help_text="Description of the server profile")
     managed = models.BooleanField(help_text="Boolean, True if the host will be managed")
```

Comment:
Shouldn't this field also have a default value?

Reply:
It doesn't seem to need one.